mllib/src/main/scala/org/apache/spark/ml/r/LinearSVCWrapper.scala

```diff
@@ -38,9 +38,17 @@ private[r] class LinearSVCWrapper private (
   private val svcModel: LinearSVCModel =
     pipeline.stages(1).asInstanceOf[LinearSVCModel]
 
-  lazy val coefficients: Array[Double] = svcModel.coefficients.toArray
+  lazy val rFeatures: Array[String] = if (svcModel.getFitIntercept) {
+    Array("(Intercept)") ++ features
```

Comment:
In R we stack ```intercept``` with other feature names, you can refer spark.glm, spark.logit, spark.survreg.